commit aa27023c941f88da383e8f2eeec2a81312243e6e
Author: Frank Ch. Eigler <fche@redhat.com>
Date:   Fri Mar 11 10:29:55 2022 -0500

    sys/sdt.h: set x86-64 STAP_SDT_ASM_CONSTRAINT back to "nor"
    
    It turns out the kernel and some other sdt consumers haven't learned
    how to use %xmm registers in sdt operands.  So under this duress, stap
    will go back to the old school integer register set "nor" as a
    default.  We'll revisit this in the future, though this egg might not
    turn into a chicken.
    
    https://bugzilla.redhat.com/show_bug.cgi?id=2058908

diff --git a/includes/sys/sdt.h b/includes/sys/sdt.h
index 48cb7263f..ca0162b4d 100644
--- a/includes/sys/sdt.h
+++ b/includes/sys/sdt.h
@@ -100,8 +100,6 @@
 # ifndef STAP_SDT_ARG_CONSTRAINT
 # if defined __powerpc__
 # define STAP_SDT_ARG_CONSTRAINT        nZr
-# elif defined __x86_64__
-# define STAP_SDT_ARG_CONSTRAINT norx
 # else
 # define STAP_SDT_ARG_CONSTRAINT        nor
 # endif
